### PR TITLE
Updates for Travis/setup.cfg

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

**Comment by JayH5** [comment]
There doesn't seem to be any coveralls for some reason :/

File: .travis.yml [hunk 1]
```diff
@@ -19,8 +19,11 @@ services:
 before_install:
   - mkdir -p $HOME/.hypothesis
   # If necessary, set up an appropriate version of pypy.
-  - if [ ! -z "$PYPY_VERSION" ]; then source setup-pypy-travis.sh; fi
-  - if [ ! -z "$PYPY_VERSION" ]; then python --version 2>&1 | fgrep "PyPy $PYPY_VERSION"; fi
+  - >
+     if [[ -n "$PYPY_VERSION" ]]; then
```
Comment (jerith) [inline .travis.yml]: Can we be sure that Travis uses `bash` or some other shell that supports `[[`? If not, it's probably safer to stick with `[` instead.

File: setup.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=test_settings
-addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8 --cov --nomigrations
+addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8
```
Comment (jerith) [inline setup.cfg]: It's useful to have coverage on local runs without having to explicitly ask for it, but I don't think there's a sane way to do that and also skip coverage on pypy builds. :-/

File: .travis.yml (hunk 1 above)
Comment (JayH5) [inline .travis.yml]: Travis's documentation is a bit sketchy in places. The best I can find is this: https://docs.travis-ci.com/user/customizing-the-build/#Note-on-%24%3F *"Each command in script is processed by a special bash function"*

File: setup.cfg (hunk 2 above)
Comment (JayH5) [inline setup.cfg]: I would probably normally agree but the tests are pretty slow even running locally and any way that they can be sped up...

Comment (jerith) [inline setup.cfg]: IIRC, it's really just the Hypothesis tests that are slow (and that's more the Django ORM's fault than anything else). Does turning off coverage give a noticeable speedup?

Comment (JayH5) [inline setup.cfg]: ofc it's a bit hard to test well. But 2 runs w/o `--cov` took ~70s on my machine. And 2 runs with `--cov` took ~100s. So I'd say it does give a noticeable speedup.

**Comment by jerith** [comment]
:+1:

**Pull request merged**